tests/test_distributed.py

```diff
@@ -169,6 +169,23 @@ def test_chunked_teacher(self):
         assert valid['exs'].value() == 100
         assert test['exs'].value() == 100
 
+    def test_no_model_parallel(self):
+        """
+        Checks that we throw an error when combining mp_train with.
+
+        --model-parallel true.
+        """
+        config = copy.deepcopy(self._base_config)
+        config['model_parallel'] = True
+        for m in [
+            'transformer/generator',
```

Comment:
Why these models only?

Reply:
These are the only models that support it, and it covers TRA, TCA, and TGA, so all the abstract classes are covered.